src/features/CourtCaseList/CourtCaseListEntry.tsx

```diff
@@ -0,0 +1,184 @@
+  const unlockCaseWithReasonPath = (reason: "Trigger" | "Exception", caseId: string) => {
+    searchParams.append(`unlock${reason}`, caseId)
+    return `${basePath}/?${searchParams}`
+  }
+  const canUnlockCase = (lockedUsername: string): boolean => {
+    return currentUser.groups.includes("Supervisor") || currentUser.username === lockedUsername
+  }
+
+  const exceptions = groupErrorsFromReport(errorReport)
+  const [showPreview, setShowPreview] = useState(false)
+  const userNotes = filterUserNotes(notes)
+  const numberOfNotes = userNotes.length
+  const mostRecentUserNote = getMostRecentNote(userNotes)
+
+  const triggerRow =
+    triggers.length > 0 ? (
+      <Table.Row className={classes.triggersRow}>
+        <Table.Cell>
+          <ConditionalRender isRendered={!!triggerLockedByUsername}>
+            <Image src={"/bichard/assets/images/lock.svg"} width={20} height={20} alt="Lock icon" />
+          </ConditionalRender>
+        </Table.Cell>
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell />
+        <Table.Cell>
+          {triggers?.map((trigger, triggerId) => (
+            <GridRow key={`trigger_${triggerId}`}>{getTriggerWithDescription(trigger.triggerCode)}</GridRow>
+          ))}
+        </Table.Cell>
+        <Table.Cell>
+          {triggerLockedByUsername && canUnlockCase(triggerLockedByUsername) ? (
+            <LockedByTag
+              lockedBy={triggerLockedByUsername}
+              unlockPath={unlockCaseWithReasonPath("Trigger", `${errorId}`)}
+            />
+          ) : (
+            <LockedByTag lockedBy={triggerLockedByUsername} />
+          )}
+        </Table.Cell>
+      </Table.Row>
+    ) : (
+      <></>
+    )
+
+  return (
+    <>
+      <Table.Row className={classes.caseDetailsRow}>
+        <Table.Cell>
+          <ConditionalRender isRendered={!!errorLockedByUsername}>
+            <Image src={"/bichard/assets/images/lock.svg"} width={20} height={20} alt="Lock icon" />
+          </ConditionalRender>
+        </Table.Cell>
+        <Table.Cell>
+          <Link href={caseDetailsPath(errorId)} id={`Case details for ${defendantName}`}>
+            {defendantName}
+            <br />
+            <ResolvedTag isResolved={resolutionTimestamp !== null} />
+          </Link>
+        </Table.Cell>
+        <Table.Cell>
+          <DateTime date={courtDate} dateFormat={displayedDateFormat} />
+        </Table.Cell>
+        <Table.Cell>{courtName}</Table.Cell>
+        <Table.Cell>{ptiurn}</Table.Cell>
+        <Table.Cell>
+          <UrgentTag isUrgent={isUrgent} />
+        </Table.Cell>
+        <Table.Cell>
+          <NotePreviewButton previewState={showPreview} setShowPreview={setShowPreview} numberOfNotes={numberOfNotes} />
+        </Table.Cell>
+        <Table.Cell>
+          {Object.keys(exceptions).map((code, codeId) => (
+            <GridRow key={`exception_${codeId}`}>
```

Comment:
Like above with the `index`.

Reply:
This was not changed as part of this ticket and is old code.

Reply:
Fair, I'm resolving this comment.